content/app/github/wrappers/dom-element-wrapper.js

```diff
@@ -47,9 +59,7 @@ define("content/app/github/wrappers/dom-element-wrapper", function(require, expo
 		 */
 		isInViewport : {
 			get : function () {
-				var clientBBox = this.clientBoundingBox;
-				var clientViewport = WindowHelper.getViewportClientBoundingBox();
-				return (clientBBox.intersection(clientViewport) !== null);
+				return (this.visibleClientBoundingBox !== null);
```

Comment:
Yes, but this way I'm isolating result calculation from return better readability.